src/components/include/utils/threads/thread.h

```diff
@@ -104,54 +104,45 @@ class Thread : public QObject {
 #else
 class Thread {
 #endif
+
+  typedef enum ThreadCommandEnum {
+    kThreadCommandNone,  // must be first
```

Comment:
@orhan-mehmedov I think that is is not good idea to name enum values in constans style. Upper case would be better.